src/components/quiz/QuizPage.tsx

```diff
@@ -2,18 +2,34 @@ import React, { useState } from "react";
 import { QuizQuestions } from "./QuizQuestions";
 import { UserDetailsForm } from "./UserDetailsForm";
 import { questions } from "./questions";
+import { Reward } from "./Reward";
+import './QuizQuestions.scss';
 
 export const QuizPage: React.FunctionComponent = () => {
   const [isUserLoggedIn, setIsUserLoggedIn] = useState(false);
-
+  const [score,setScore] = useState<number>(0);
```

Comment:
Formatting:
```suggestion
  const [score, setScore] = useState<number>(0);
```
and you don't need to specify type if it can be inferred from the value:
```suggestion
  const [score,setScore] = useState(0);
```